core/procedure.cc

```diff
@@ -38,6 +38,7 @@ ProcedureRunConfigDescription()
     nullAccepted = true;
 
     addField("id", &ProcedureRunConfig::id, "ID of run");
+    addField("state", &ProcedureRunConfig::state, "State of the run");
```

Comment:
This shouldn't be here.  It's not a configuration parameter.  That's why I don't think we should POST to the same route.